dist/less/saas-list.less

```diff
@@ -26,10 +32,12 @@
       @media (min-width: @screen-sm-min) {
         border-bottom: none;
         flex: 0 1 auto;
```

Comment:
I believe if you set flex-grow to 1 (flex: 1 1 auto), you don't need the min-widths (here and line 40).